model/src/pyrenew/latent/infection_seeding_process.py

```diff
@@ -34,6 +44,13 @@ def __init__(
         self.I_pre_seed_rv = I_pre_seed_rv
         self.infection_seed_method = infection_seed_method
         self.name = name
+        if t_start is None:
+            t_start = -infection_seed_method.n_timepoints
```

Comment:
Are we sure we want this default? CC @damonbayer

Reply:
Leaning toward yes. I think this is in line with my previous comment about this being an unenforced convention.

Reply:
Agree that if "seeding" is going to default to any time period, the period ending at $t=0$ is the right one.

Related question: we agreed to measure `t_start` in fundamental units, not array units. Are `InfectionSeedingProcess`es and/or `InfectionSeedMethod`s going to be allowed to have non-fundamental units? If so, we potentially need a unit conversion operation here. If not, we need to enforce `t_unit = 1` for both.

My take: 
- it is worth allowing them to have non-fundamental units
- It is possibly worth enforcing that an `InfectionSeedingProcess` has _the same_ units as its `InfectionSeedMethod`

Reply:
Good point. Created an issue to discuss this.